packages/math-input/src/components/keypad/mobile-keypad-internals.tsx

```diff
@@ -186,18 +186,16 @@ class MobileKeypadInternals
                     transitionLeaveTimeout={AnimationDurationInMS}
                     transitionStyle={{
                         enter: {
-                            transform: "translate3d(0, 100%, 0)",
-                            transition: `${AnimationDurationInMS}ms ease-out`,
+                            transform: "translate3d(0, 0, 0)",
```

Comment:
I'm confused. This kind of just looks like it's going from (0, 0, 0) to (0, 0, 0).